src/main/java/uk/gov/hmcts/reform/sscs/bulkscancore/handlers/CcdCallbackHandler.java

```diff
@@ -38,15 +46,30 @@ public CcdCallbackResponse handle(
         // Transform into SSCS case
         CaseTransformationResponse caseTransformationResponse = caseTransformer.transformExceptionRecordToCase(exceptionRecordData);
 
+        Map<String, Object> transformedCase = caseTransformationResponse.getTransformedCase();
+
         if (!ObjectUtils.isEmpty(caseTransformationResponse.getErrors())) {
             return CcdCallbackResponse.builder()
-                .errors(caseTransformationResponse.getErrors())
                 .data(exceptionRecordData)
+                .errors(caseTransformationResponse.getErrors())
+                .build();
+        }
+
+        // Validate the transformed case against robotics schema
```

Comment:
what's this comment about?

Reply:
This is specific to SSCS validation

Reply:
I'm not too familiar with this code, but this looks to be in a generic area for the library?
so shouldn't have anything sscs specific in it?